recursio-server/GameRoom/ServerGhostManager.gd

```diff
@@ -62,14 +70,15 @@ func _use_new_record_data():
 
 
 # OVERRIDE ABSTRACT #
-func _on_ghost_hit(perpetrator, victim):
-	_new_previous_ghost_death.append(_create_new_ghost_death_data(victim, perpetrator))
-	emit_signal("ghost_hit",victim.player_id, victim.timeline_index, perpetrator.player_id, perpetrator.timeline_index)
+func _on_ghost_hit(hit_data: HitData):
+	_new_previous_ghost_death.append(_create_new_ghost_death_data(hit_data))
+	# TODONOW: follow this thread
```

Comment:
What's Todo now?

Reply:
whoops that's just a temporary comment so I didn't loose track during the refactor, removed it.